src/shared/components/Button/Button.style.ts

```diff
@@ -72,53 +75,116 @@ const colorsFromProps = ({ variant }: ButtonStyleProps) => {
   return styles
 }
 
-const sizeFromProps = ({ size = 'regular', full, hasText }: ButtonStyleProps) => {
-  let padding, fontSize
+const sizeFromProps = ({ size = 'medium', full, hasText }: ButtonStyleProps) => {
+  let padding, fontSize, buttonSizeWithoutText
   switch (size) {
-    case 'smaller': {
-      padding = '10px'
+    case 'small': {
+      padding = '9.5px 12px'
```

Comment:
I don't think we need this additional `.5px`? The button seems to have the correct size (`32px`) with `9px` vertical padding